hphp/runtime/vm/jit/abi-arm.cpp

```diff
@@ -161,13 +161,20 @@ PhysReg rarg_simd(size_t i) {
   assertx(i < num_arg_regs_simd());
   return vixl::FPRegister::DRegFromCode(i);
 }
+PhysReg rarg_ind_ret(size_t i) {
+  assertx(i < num_arg_regs_ind_ret());
+  return vixl::x8;
```

Comment:
All supported platforms on HHVM have zero or one indirect return register, so we don't need an index argument for this.